README.txt

```diff
@@ -1 +1,24 @@
-## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
\ No newline at end of file
+## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
+
+**How to Earn XP**
+
+**Earn Activity XP**
+
+- As a lender, provide your first loan (+1XP for every 5th)
+- As a borrower, pay interest on your first loan (+1XP for every 5th)
+- Approve your first lending strategy (+1XP for every 5th)
+- Lend or borrow the highest loan amount in a given month
+
+**Earn Contributor XP**
+
+- Leave your first constructive comment on Github (+1XP for every 5th)
+- Design assets or feature concepts used in production (+1XP for every 5th)
+- Contribute meaningful code on Github (+1XP for every 5th)
+- Provide meaningful feedback from testnet or early release testing
+
+**Earn Community XP**
+
+- Join Backed Discord and share good feedback or a new meme
```

Comment:
I think the more X-for-Y we can list the better, since it'll be easier to entice people to engage if we give them specific challenges. But I'll take a pass at example self-nominations, maybe in Notion so we can grind it out line by line.

Reply:
> agree the more x-for-y the better

with the caveat that I think they should be clear cut and not require a lot of human discretion? Also think OK to launch with some minimal list and rely on the community to vote in more?